Blog: Install Files with WinGet (#730)
files-community/Website: src/routes/blog/posts/now-on-winget/+page.md

---

> title: Files is now available on WinGet again

Reviewer: ```suggestion
title: Files is back on WinGet
```

> title: Files is now available on WinGet again

Reviewer: ```suggestion
title: Files is now available on WinGet
```

description: Files is back on WinGet!
thumbnail: /blog-resources/now-on-winget/Hero.png
date: 11/30/2024
author: Lamparter
---

I'm excited to announce that Files is now available again on WinGet! After a couple years, we're pleased to bring back this popular installation method for Files.

### Files and WinGet

The return of Files to WinGet signifies our continued commitment to providing users with the best experience, allowing them to install and manage their apps the way they desire.

> ### How to Install Files via WinGet

Reviewer: ```suggestion
### How to install Files via WinGet
```

---
title: Files is now available on WinGet again
description: Files is back on WinGet! I'm pleased to announce that Files can finally be installed from WinGet again.
thumbnail: /blog-resources/now-on-winget/Hero.jpg
date: 11/17/2024
author: Lamparter
---

> I'm are excited to announce that the app Files is now available again on WinGet! After initially being removed, Files has made a triumphant return, offering users an enhanced and seamless file management experience on Windows.

Reviewer: ```suggestion
I'm excited to announce that Files is now available again on WinGet! After a couple years, we're pleased to bring back this popular installation method for Files.
```

### Files and WinGet

> The return of Files to WinGet signifies the continued commitment to providing users with the best experience in offering convenient, friendly installation methods compatible with their device.

Reviewer: ```suggestion
The return of Files to WinGet signifies our continued commitment to providing users with the best experience, allowing them to install and manage their apps the way they desire.
```

> WinGet allows users to easily install, update, and manage software packages on their Windows devices. By integrating Files into WinGet, users can now enjoy a hassle-free installation process and keep their file manager up to date with the latest features and improvements.
> 

Reviewer: ```suggestion
```
